python-sdk/hatchet_sdk/clients/events.py

```diff
@@ -28,10 +35,22 @@ def push(self, event_key, payload):
         request = PushEventRequest(
             key=event_key,
             payload=payload_bytes,
-            eventTimestamp=timestamp_pb2.Timestamp().FromDatetime(datetime.datetime.now()),
+            eventTimestamp=proto_timestamp_now(),
         )
 
         try:
             self.client.Push(request, metadata=get_metadata(self.token))
         except grpc.RpcError as e:
             raise ValueError(f"gRPC error: {e}")
+    
+    def log(self, message: str, step_run_id: str):
+        try:
+            request = PutLogRequest(
+                stepRunId=step_run_id,
+                createdAt=proto_timestamp_now(),
+                message=message,
+            )
+
+            self.client.PutLog(request, metadata=get_metadata(self.token))
```

Comment:
is this sync in python?

Reply:
Good catch. I'll use a thread pool executor for the event client logger.